main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
I can't see any problem here. In go, your tests must go in the same package for that exaclty reason, so you have access to private stuff. See [here](http://golang.org/pkg/testing/)

![image](https://cloud.githubusercontent.com/assets/7764293/9707491/bbac0a58-54e0-11e5-839b-9ffdb84af7d2.png)